tests/src/test/resources/application.conf

```diff
@@ -5,6 +5,17 @@ tests {
   clean-up = false
 }
 
+akka {
+  http {
+    sse {
+      # The maximum size for parsing server-sent events (96KiB).
+      max-event-size = 98304
+      # The maximum size for parsing lines of a server-sent event (48KiB).
+      max-line-size = 49152
```

Comment:
Due to the change in the search config, the default search view resource gets bigger, and could not be streamed as sse without this config change.